scripts/install-linux-deps.sh

```diff
@@ -56,4 +56,4 @@ env DEBIAN_FRONTEND=noninteractive apt-get install --yes --quiet \
   && rm -rf /var/lib/apt/lists/*
 
 # for boost
-pip install conan
+python3 -m pip install conan
```

Comment:
imho: more secure with `--upgrade` 
* `python3 -m pip install --upgrade conan`

so we can remove the same conan upgrade lines from the new "Dockerfile"  ( added by me )